package.json

```diff
@@ -40,6 +40,8 @@
     "@mdx-js/tag": "^0.20.3",
     "babel-plugin-tailwind": "^0.1.10",
     "classnames": "^2.2.6",
+    "core-js": "^2.6.5",
```

Comment:
Do we need this?

Reply:
I can't run local server without it somehow.